classes/class-dintero-checkout-confirmation.php

```diff
@@ -113,7 +113,7 @@ public function handle_error( $error, $order ) {
 			wc_add_notice( $note, 'error' );
 		}
 
-		Dintero_Checkout_Logger::log( "REDIRECT ERROR [$error]: $note WC order id: $order_id / %s: %s", $note, $order_id );
+		Dintero_Checkout_Logger::log( "REDIRECT ERROR [$error]: $note WC order id: $order_id / %s: %s" );
```

Comment:
Wont this result in %s: %s being added to the string still at the end? Or is there a sprintf down the chain?